src/components/QRShare/index.js

```diff
@@ -82,7 +76,7 @@ class QRShare extends Component {
                         family="EXP_NEUE_BOLD"
                         fontSize={13}
```

Comment:
Same here

Reply:
Which variable should I use for this one? May be this one?     
```
fontSizeLabel: getValueUsingPixelRatio(13, 19),
```

Reply:
Yep `fontSizeLabel` looks like the right one.

Reply:
Cool. I'll update it in couples of minutes